changelog/updates/2023-02-06-weekly-updated.md

```diff
@@ -0,0 +1,10 @@
+- XZ utils ([5.4.1](https://github.com/tukaani-project/xz/releases/tag/v5.4.1) (includes [5.4.0](https://github.com/tukaani-project/xz/releases/tag/v5.4.0)))
+- vim ([9.0.1157](https://github.com/vim/vim/releases/tag/v9.0.1157))
+- SDK: qemu ([7.2.0](https://wiki.qemu.org/ChangeLog/7.2))
+- SDK: boost ([1.81.0](https://www.boost.org/users/history/version_1_81_0.html))
+- bpftool ([5.19.12](https://lwn.net/Articles/838789/))
```

Comment:
That link points to Kernel 5.9.12, a wrong version.
The correct one is actually https://lwn.net/Articles/909678/.

Reply:
Oh, oops. Thanks for spotting, fixed.